src/arch/xtensa/init.c

```diff
@@ -36,7 +36,9 @@ uint32_t lock_dbg_atomic;
 /** \brief Debug locks per user. */
 uint32_t lock_dbg_user[DBG_LOCK_USERS] = {0};
 #endif
-
+#ifndef CONFIG_HAVE_SLAVE_CORE_ROM
+SHARED_DATA void *g_shared_vecbase_ptr;
```

Comment:
Why `g_` pferix here? I guess it's for  global, but other symbols don't use it in our repo.

Reply:
sure, changed to shared_vecbase_ptr